lib/records/record.rb

```diff
@@ -109,7 +118,8 @@ def careplan_start(type, activities, time, reason)
           'type' => type,
           'activities' => activities,
           'time' => time,
-          'reason' => reason
+          'start_time' => time,
+          'reasons' => reason
```

Comment:
Might want to rename `reason` argument to `reasons` (for consistency and to avoid unintended misuse of the API).